app/assets/javascripts/sitewide/device_revocation.js

```diff
@@ -0,0 +1,5 @@
+$(document).on('turbolinks:load', function() {
+  $(document).on('click', '#revokeDeviceBtn', function() {
+    $('#deviceRevocationModal').modal('show');
+  });
+});
```

Comment:
Do we need this? I think for just opening the modal, we can use:
`data-toggle="modal" data-target="#revokeDeviceBtn"`
on the button.